### PR TITLE
 feat(#212) Navbar Component

### LINKED ISSUE #212: Navbar Component
### What needs to be done
- Create a static navbar component along with dynamic sub section for navbar. 
- All nav links should redirect to a certain page. 
- The sub section of the navbar should change based on the main navbar link that is (see design in #). 
- The navbar should match the styling and branding of the website. 
- Data may need to change depending on the role (what they are allowed to do / permissions) 

### Out of scope
Creating the pages that are linked in the navbar

### PR DESCRIPTION
**Pull request by joeyabou**
**Navbar** 

Added the static Navbar component as well as the dynamic subnavbar. Also, implemented the ability to hide any of the chosen navbar components depending on if the user has access to those functions. 

### Test plan
image

### Closing issues
List all issues this PR resolves.
- Closes #212 

### Pre-Merge checklist 
Complete these items before requesting reviews
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions 

### Sub-tasks Checklist
- [x] UI Implementation
- [x] UI Test
- [ ] Backend Implementation
- [ ] Backend Tests
- [ ] Integration Tests
- [ ] System Tests

### CODE REVIEW

**Review by jonathanp16 — APPROVED** [review]
Overall looks very solid. I like the use of binding certain classes based on an active boolean, as well as the way you restructured the Jetstream Navlink to fit our needs. Good coverage and no code smells. Good job!

**Review by danseremet — APPROVED** [review]
LGTM!

**Review by jonathanp16 — CHANGES_REQUESTED** [review]
Looks like there's a merge conflict in the User model

File: app/Models/User.php [hunk 1]
```diff
@@ -59,8 +59,14 @@ class User extends Authenticatable
      */
     protected $appends = [
         'profile_photo_url',
+        'can',
```
Comment (WindOfRussia) [inline app/Models/User.php:62]: by appending the dynamic attribute, we ensure its going to be always available in the frontend user object

**Review by WindOfRussia — DISMISSED** [review]
re-approving this as the underlying conflict was resolved

**Review by jonathanp16 — DISMISSED** [review]

**Review by alexstojda — DISMISSED** [review]
The navbar does not seem to be working as show in the screenshots, see the demo deployment linked to the PR

Additionally please use the application name & logo define in the settings, see https://github.com/CSU-Booking-Platform/application/pull/232

image

**Review by Alex-Hayeur — DISMISSED** [review]
The logo is still incorrect as-per Alex's review, his review was discarded by mistake.

**Review by alexstojda — APPROVED** [review]
Thanks for implementing the changes, looks ok to me now.

**Review by jonathanp16 — DISMISSED** [review]

**Comment by alexstojda** [comment]
having a look at the merge issues

**Review by alexstojda — DISMISSED** [review]

**Review by jonathanp16 — DISMISSED** [review]

**Review by alexstojda — APPROVED** [review]
ok merge issue should be resolved

**Review by jonathanp16 — APPROVED** [review]

**Pull request merged**